package.json

```diff
@@ -54,7 +54,7 @@
     "tsml": "1.0.1",
     "videojs-font": "2.1.0",
     "videojs-ie8": "1.1.2",
-    "videojs-vtt.js": "0.12.4",
+    "videojs-vtt.js": "0.12.5",
```

Comment:
there are no functional changes in this release, it just adds another build output.